R/between.R

```diff
@@ -4,22 +4,45 @@ between <- function(x,lower,upper,incbounds=TRUE) {
   if (is.logical(lower)) lower = as.integer(lower)   # typically NA (which is logical type)
   if (is.logical(upper)) upper = as.integer(upper)   # typically NA (which is logical type)
   is_strictly_numeric <- function(x) is.numeric(x) && !inherits(x, "integer64")
+  try_posix_cast <- function(x, tz) {
+    x_time = as.POSIXct(x, tz = tz)
+    if (is.na(x_time)) return(x)
```

Comment:
This line is missing coverage according to codecov.  Needs to be covered before merge please.

Reply:
tests for exceptions added

Reply:
thanks @jangorecki. anything else needed for the code i added?